Orange/widgets/settings.py

```diff
@@ -1235,6 +1235,8 @@ def _fix(name):
         var, vartype = settings.values[name]
         if 0 <= vartype <= 100:
             settings.values[name] = (var, 100 + vartype)
+        elif var == none_placeholder and vartype == -2:
+            settings.values[name] = None
```

Comment:
@astaric, could you check this? `DomainModel` and `VariableModel` do not contain a string placeholder `"(None)"` but an actual `None`. Hence migration from string settings to variables, must replace `"(None)"` with `None`.

I however wonder why haven't we noticed this problem before, for instance in scatter plot's shape combo. Is Mosaic somehow specific? Or is the bug somewhere else and this fix somehow masks it?